### PR TITLE
ROC-5615 remove ccj upload  and download code

### PR DESCRIPTION
**Pull request by srineelesh**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5615

### Change description ###
removed code to upload and download ccj document

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
```

### CODE REVIEW

File: src/aat/resources/environment.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pdf_service.url = http://dummy-url
+pdf_service.url = http://localhost:5500
```
Comment (johnhosolirius) [inline src/aat/resources/environment.properties]: You sure this'll work on AAT?

Comment (srineelesh) [reply src/aat/resources/environment.properties]: no :) .. have fixed this.

File: src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BasePdfTest.java [hunk 2]
```diff
@@ -40,12 +45,8 @@ private InputStream retrievePdf(String pdfName, String externalId) {
...
     }
 
-    protected InputStream retrieveCCJPdf(String externalId) {
-        return RestAssured
-            .given()
-            .header(HttpHeaders.AUTHORIZATION, user.getAuthorisation())
-            .get("/documents/ccj/" + externalId)
-            .asInputStream();
+    protected InputStream retrieveCCJPdf(Claim claim) {
+        return new ByteArrayInputStream(countyCourtJudgmentPdfService.createPdf(claim));
```
Comment (dharmendrak) [inline src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BasePdfTest.java]: Any particular reason we are using `countyCourtJudgmentPdfService` to create Pdf, as in all functional test we are trying to call individual apis.

Comment (dharmendrak) [reply src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BasePdfTest.java]: @ckalista @srineelesh Any reason we have polluted AAT tests with claim store services as are trying to test and use just end points here.

**Pull request merged**